CHANGELOG.md

```diff
@@ -22,6 +22,12 @@ Will become:
 }
 ```
 
+### Config Changes
+
+- **Breaking Change** The `plugin:shopify/react` is no longer a core config and must augment one of the `plugin:shopify/typescript` or `plugin:shopify/esnext` configs
```

Comment:
Will people know what this even means for them? Can we show them an example config diff?